Update TsFileOutputFormat.java

asia-zengtao opened the pull request:
Exception handling is improved by providing specific error messages when dealing with URISyntaxException

asia-zengtao committed:
Update TsFileOutputFormat.java

Exception handling is improved by providing specific error messages when dealing with URISyntaxException

asia-zengtao committed:
Update pom.xml

sonarcloud

asia-zengtao committed:
Create build.yml

sonarcloud

HTHou commented on .github/workflows/build.yml:
Consider remove this workflow file?

HTHou commented on pom.xml:
```diff
@@ -124,6 +124,8 @@
     </modules>
     <!-- Properties Management -->
     <properties>
+        <sonar.organization>asia-zengtao</sonar.organization>
+        <sonar.host.url>https://sonarcloud.io</sonar.host.url>
```
remove it?

asia-zengtao replied:
Ok I will remove it! thanks !

asia-zengtao committed:
Merge branch 'apache:master' into master

asia-zengtao committed:
Update build.yml

sonar

asia-zengtao committed:
Update build.yml

asia-zengtao committed:
remove sonarcloud config info

remove sonar config info
<sonar.organization>asia-zengtao</sonar.organization>
<sonar.host.url>https://sonarcloud.io</sonar.host.url>

asia-zengtao committed:
Update build.yml

remote sonar cloud build.yml config info

HTHou commented on .github/workflows/build.yml:
Empty file?
